pkg/downloader/downloader.go

```diff
@@ -385,3 +392,86 @@ func downloadHTTP(localPath, url string, expectedDigest digest.Digest) error {
 
 	return nil
 }
+
+// Read the file and return its contents as a string
+// Never use it with large files
+func readSmallTextFile(filename string) (string, error) {
+	data, err := os.ReadFile(filename)
+	if err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
```

Comment:
Not worth defining as a new function

Reply:
But it's called from 3 places in my code.. Could you please suggest what is the solution then?

Reply:
Ah.. you're right actually.. there are just 2 invocations left and I think it should be easy to deal with